downloader/src/main/kotlin/vcs/Mercurial.kt

```diff
@@ -65,7 +65,7 @@ class Mercurial : VersionControlSystem() {
     override fun getWorkingTree(vcsDirectory: File) = MercurialWorkingTree(vcsDirectory, type)
 
     override fun isApplicableUrlInternal(vcsUrl: String) =
-        ProcessCapture("hg", "identify", vcsUrl).isSuccess
+        MercurialCommand.isInPath() && ProcessCapture("hg", "identify", vcsUrl).isSuccess
```

Comment:
If we consider adding this fault tolerance the right thing to do, should the check then be moved a bit upwards the call tree so that it becomes general fall all `VersionControlSystem` implementations?